lib/seadsa/TypeUtils.cc

```diff
@@ -58,9 +60,11 @@ void AggregateIterator::doStep() {
       continue;
     }
 
-    if (Ty->isVectorTy()) {
-      for (size_t i = 0, e = Ty->getVectorNumElements(); i != e; ++i)
-        Worklist.push_back(Ty->getVectorElementType());
+    // Shaobo: LLVM 11 introduced `ScalableVectorType` whose number of elements
```

Comment:
this means that `ScalabeVectorType` is not handled. At least add an assert so that we can catch it in debug builds

Reply:
Sounds good.